spec/lib/loader_spec.rb

```diff
@@ -39,7 +39,7 @@
     end
 
     it 'expects all generated roles to have the type \'Role\'' do
-      expect(Role.select(:type).all.map(&:type).uniq).to match_array ['Role']
+      expect(Role.pluck(:type).uniq).to match_array ['Role']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.